crates/sui-indexer-alt/src/handlers/sum_coin_balances.rs

```diff
@@ -99,9 +99,17 @@ impl Processor for SumCoinBalances {
                     continue;
                 };
 
-                // Coin balance only tracks address-owned objects
-                let Owner::AddressOwner(owner_id) = object.owner() else {
-                    continue;
+                // Coin balance only tracks address-owned or ConsensusV2 objects
+                let (owner_kind, owner_id) = match object.owner() {
+                    Owner::AddressOwner(owner_id) => (StoredOwnerKind::Address, owner_id),
+                    // ConsensusV2 objects are treated as address-owned for now in indexers.
+                    // This will need to be updated if additional Authenticators are added.
+                    Owner::ConsensusV2 { authenticator, .. } => (
+                        StoredOwnerKind::ConsensusV2,
```

Comment:
We should use a different enum here to the one we use for objects in general, because while we don't want to distinguish consensus-managed single-owned objects from address-owned objects, we do want to do that for coins.

Reply:
created a new enum